### DISCUSSION TITLE
3.4.0rc2

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1140] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.4.0rc2'>3.4.0rc2</a>.</em>

**Comment by snowman2** [comment]
The plan is to release with PROJ 9.1 in the wheels.

Release Candidate Status:

- [x] Linux ARM wheels (https://github.com/pyproj4/pyproj-wheels/pull/54)
- [ ] MacOS ARM wheels (@vot4anto)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/122)
- [x] Debian (@sebastic)
- [ ] Fedora (@jdkloe)

Plan to release over the weekend. Testing is always appreciated, but definitely not required.

**Reply by sebastic** [reply]
RC2 is looking good: https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental
